docs/modules/ROOT/pages/reference/service-requirements.adoc

```diff
@@ -0,0 +1,88 @@
+= Service Requirements and Iterations
+
+Implementing and maintaining a service is iterative work.
+The idea is to release early and release often, without having the urge to have all features available right from the beginning.
+
+This page gives an idea on the iterative steps.
+Except the initial version and the feature complete version, all steps in-between can be in any order, as it fits best.
+With "feature complete" it's not meant that the service doesn't receive new functionality or doesn't get maintained anymore, we're talking about the default scope outlined on this page.
```

Comment:
I have a question regarding the development of the framework itself.
It's my understanding that the framework should help the services covering the iterations outlined below. By extension it means the framework must be capable of supporting these iterations, correct?
Do we know develop or choose a technology that can cover all of this from the beginning, or will there be a complex mix of various technology stack to achieve these requirements with the framework as they come up?

Reply:
Not sure yet, I have to think about that point...